ntbs-service/Pages/Notifications/OverviewPartials/_ClinicalDetailsOverviewPartial.cshtml

```diff
@@ -91,10 +91,7 @@
     </nhs-grid-row>
 
     <nhs-grid-row>
-        <nhs-grid-column grid-column-width="OneQuarter">
-            <div class="notification-details-label"> Date of death </div>
-            <div class="cell-min-height"> @Model.Notification.FormattedDeathDate </div>
-        </nhs-grid-column>
+        <div> Do we want to pull post mortem in to  </div>

```

Comment:
Not sure if this is an old bit of diff, but don't want to lose my place on the page.